README.adoc

```diff
@@ -104,8 +104,10 @@ The [hotspot=43]`@ArquillianResource` annotation is used to retrieve the
 `\http://localhost:9080/arquillian-managed/` base URL for this web service.
 The annotation provides the host name, port number and web archive
 information for this service, so you don't need to hardcode these values
-in the test case. Notice the `arquillian-managed` WAR name that was specified
-in the web archive, it's needed  when the `inventory` service communicates
+in the test case. The `arquillian-managed` WAR name comes
+from the WAR name you specified when you created the web archive in the
+[hotspot=36-41]`@Deployment` method.
+It's needed  when the `inventory` service communicates
```

Comment:
Extra space between 'needed' and 'when'?  A nit.

Reply:
Fixed